_i18n/de.yml

```
@@ -0,0 +1,90 @@
+meta:
+  description: Entwickle 2D- und 3D-Spiele, plattformübergreifenden Projekte oder sogar XR-Ideen!
+header:
+  features: Features
+  showcase: Showcase
```

```suggestion
  showcase: Galerie
```
Alternatives: Spiele-Galerie, Ausstellung, Spiele-Sammlung